content/en/tracing/custom_instrumentation/dotnet.md

```diff
@@ -17,9 +17,11 @@ further_reading:
       text: 'Explore your services, resources, and traces'
 ---
 <div class="alert alert-info">
-If you have not yet read the instructions for auto-instrumentation and setup, start with the <a href="https://docs.datadoghq.com/tracing/setup/dotnet/">.NET Setup Instructions</a>.
+If you have not yet read the instructions for auto-instrumentation and setup and are looking for those, you can find them here: <a href="https://docs.datadoghq.com/tracing/setup/dotnet/">.NET Setup Instructions</a>.
```

Comment:
How about something like:

```suggestion
For instructions on how to setup the .NET Tracer and enable automatic instrumentation, see the [.NET Setup Instructions][???].
```

(note also I changed the link syntax, but it needs adding the link to the list below).